c10/core/DispatchKey.h

```diff
@@ -444,6 +444,8 @@ enum class DispatchKey : uint16_t {
   Autograd,
   CompositeImplicitAutograd, // registered at
   // build/aten/src/ATen/RegisterCompositeImplicitAutograd.cpp
+  CompositeImplicitBatched, // registered at
+  // build/aten/src/ATen/RegisterCompositeImplicitBatched.cpp
```

Comment:
note: if mobile or ovrsource ever end up using vmap internally, you'll need to update internal to tell buck about the new codegen'd file. My guess is that neither of them use vmap today though? @zou3519.

Reply:
I don't think either of them do, but I'll import this to check it out, thanks for the head's up